### ISSUE TITLE
Show excerpt/preview of contents in file list pane

### ISSUE THREAD

**Issue by jsit** [fabiospampinato/notable#244]
Would love to see a preview of the first 2-3 lines of a document in the file list pane (the middle pane).

**Comment by fabiospampinato** [comment]
This is a feature pretty much all other note-taking apps have. 

I disabled this on the app I was using previously so I also didn't implement this for first few releases of the app.

Let's monitor the popularity of the request, I might add configurable support for this.

**Comment by absir1949** [comment]
i think it can be previewed as a card.

**Comment by bendem** [comment]
Yeah, I'd like the possibility of having a card view with more information (note tags, attachment count, excerpt, creation/update date) available at a glance.

**Comment by deanbarrow** [comment]
As well as tags, it would be great to know which notebook the note belongs to.
In my use case I might have a 'TODO' named note in different project notebooks so if I search 'todo' then right now the results aren't that helpful.
